insights/parsers/satellite_postgresql_query.py

```diff
@@ -139,6 +148,7 @@ class SatelliteAdminSettings(SatellitePostgreSQLQuery):
         >>> table.get_setting('destroy_vm_on_host_delete')
         True
     """
+    headers = ['name', 'value', 'default']
```

Comment:
This is different with the description of Line-163

Reply:
Do u mean I don't mention "name" in Line-163? In Line-163, I just explain "default" and "value" must both be queried to determine the final value of the setting.

Reply:
By defining the `header`, I think it implicitly means that the header line of the content should contain all of the items in the `header`.  so the L145~L146 is not required anymore.

And the base class should also be updated as:

```
if not all(t in content[start_index] for t in self.headers):
   raise ...
```

Reply:
I just enhanced the `calc_offset` and added a new `require_all` parameter
- https://github.com/RedHatInsights/insights-core/pull/3316
After that, you can use it, like:
```
start_index = calc_offset(content, self.headers, require_all=True)
```